### PR TITLE
Add page x of y to title tag

### PR DESCRIPTION
**Pull request by nicola-d-bbc**
Resolves [WSTEAMA-49](https://jira.dev.bbc.co.uk/browse/WSTEAMA-49)

**Overall change:**
Added active page, current page and brand name to title tag of topic page, and refactored getTranslations out of Pagination.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]
Looks good 👍 are we sure this is the correct element though?  Reading that ticket, it sounds like it might be referring to the `<title>` element, rather than the page's `<h1>`

**Comment by nicola-d-bbc** [comment]
> Looks good 👍 are we sure this is the correct element though? Reading that ticket, it sounds like it might be referring to the `<title>` element, rather than the page's `<h1>`

ahh you might be right... let me check 😅

File: src/app/pages/TopicPage/TopicPage.jsx [hunk 1]
```diff
@@ -36,12 +36,30 @@ const TopicPage = ({ pageData }) => {
...
+
+  const pageTitle = `${title}, ${translatedPage}`;
+
   return (
     <Wrapper role="main">
       <ATIAnalytics data={pageData} />
       <ChartbeatAnalytics data={pageData} />
       <MetadataContainer
-        title={title}
+        title={activePage >= 2 && pageCount > 1 ? pageTitle : title}
```
Comment (RetiredBlob) [inline src/app/pages/TopicPage/TopicPage.jsx]: `pageCount` is always >= `activePage`
```suggestion
        title={activePage >= 2 ? pageTitle : title}
```

**Review by RetiredBlob — APPROVED** [review]
👍

**Comment by greenc05** [comment]
If I'm on page 2 of 10 for a topic of 'Donald Trump' on Pidgin, what does the title output as @nicola-d-bbc ?

**Review by pvaliani — APPROVED** [review]
As we are using types in the BFF, should we start considering using TypeScript client side for the new pages like TopicPage to get reduce the use of prop-types?

@andrewscfc

**Comment by nicola-d-bbc** [comment]
> If I'm on page 2 of 10 for a topic of 'Donald Trump' on Pidgin, what does the title output as @nicola-d-bbc ?

Donald Trump, Page 2 of 10 - BBC News Pidgin

**Pull request merged**